python/tvm/relay/frontend/mxnet.py

```diff
@@ -646,6 +646,11 @@ def _mx_arange(inputs, attrs):
     return _op.arange(**new_attrs)
 
 
+# pylint: disable=unused-argument
+def _mx_make_loss(inputs, attrs):
```

Comment:
Just curious, is make_loss used in any inference scenario?

Reply:
Ideally, it should not be used for inference. But we received a model which was using make_loss operator from one of the customers.

Reply:
In this case, I suggest we add a comment saying something like "In inference mode make_loss is just identity and has no actual effect."

Reply:
Sure. Added the comment.